docs/extend/plugins.md

```diff
@@ -1,87 +1,272 @@
 <!--[metadata]>
 +++
-title = "Extending Engine with plugins"
-description = "How to add additional functionality to Docker with plugins extensions"
-keywords = ["Examples, Usage, plugins, docker, documentation, user guide"]
+aliases = [
+"/engine/extend/"
+]
+title = "New Plugin System"
```

Comment:
@sfsmithcha can you chime in ? I know you wanted to keep the same file to not create broken links, but I think @anusha-ragunathan has a good point.